src/Servers/Kestrel/tools/CodeGenerator/TransportStreamFeatureCollection.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeGenerator
+{
+    public class TransportStreamFeatureCollection
```

Comment:
TransportStream(FeatureCollection|Generated)?.cs look identical to the TransportConnection and TransportMultiplexedConnection counterparts except for the name.

Why not have QuicStreamContext and QuicConnectionContext just derive from TransportConnection like before? This is *a lot* of duplication.

Reply:
This duplication is now removed because I dropped StreamContext.